docs/user/exercises/programming-integrated-code-lifecycle-repository-view.inc

```diff
@@ -0,0 +1,73 @@
+For the programming exercises, we use a version control system called **Git**. Git is a distributed version control system that allows you to track changes in your code.
+These changes are stored in a repository. A repository is a collection of files and directories that are tracked by Git.
+How to navigate through the repository and view the code is explained in the following sections. This guide is divided into two parts: one for students and one for instructors.
+It is demonstrated using course programming exercises, but the same principles apply to exams as well.
+
+For Students
+""""""""""""
+
+As a student, you can use the repository to view your course submissions.
+To access your submissions, you need to navigate to the exercise and press the '*Open Repository*' button.
+
+.. figure:: programming/open-repository-button.png
```

Comment:
I don't think a big image of just the button helps. It would be better to show the whole exercise view as an example, and highlight the button. This way you can see where it is placed.